Altis_Life.Altis/core/functions/fn_revealObjects.sqf

```diff
@@ -8,8 +8,6 @@
     
     Can be taxing on low-end systems or AMD CPU users.
 */
-if (!life_settings_revealObjects) exitWith {};
```

Comment:
I like this check - I've no idea if it's ever possible to reach this point, but avoiding the next part is preferable, nothing wrong with sanity

Reply:
This check is pointless because as we should remove the EachFrame handler, instead of having the handler always on, and using a variable to stop it on line 1